showcase/plot/heatmap-chart.js

```diff
@@ -18,32 +18,42 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import React from 'react';
+import React, {Component} from 'react';
 
-import {XYPlot, XAxis, YAxis, HeatmapSeries} from 'index';
+import {XYPlot, XAxis, YAxis, HeatmapSeries, Hint} from 'index';
 
-export default function Example(props) {
-  return (
-    <XYPlot width={300} height={300}>
-      <XAxis />
-      <YAxis />
-      <HeatmapSeries
-        className="heatmap-series-example"
-        data={[
-          {x: 1, y: 0, color: 10},
-          {x: 1, y: 5, color: 10},
-          {x: 1, y: 10, color: 6},
-          {x: 1, y: 15, color: 7},
-          {x: 2, y: 0, color: 12},
-          {x: 2, y: 5, color: 2},
-          {x: 2, y: 10, color: 1},
-          {x: 2, y: 15, color: 12},
-          {x: 3, y: 0, color: 9},
-          {x: 3, y: 5, color: 2},
-          {x: 3, y: 10, color: 6},
-          {x: 3, y: 15, color: 12}
-        ]}
-      />
-    </XYPlot>
-  );
+export default class HeatmapChart extends Component {
```

Comment:
Can't wait until we're able to use hooks instead of making a class

Reply:
👍👍👍👍👍👍👍👍👍👍👍